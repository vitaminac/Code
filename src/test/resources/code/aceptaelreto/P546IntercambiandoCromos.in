2
4
2 1 2 3
5
3 3 4 4 5
3
1 2 3
5
6 6 4 4 5
