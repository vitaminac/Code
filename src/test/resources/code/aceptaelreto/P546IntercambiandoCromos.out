2
4
Nada que intercambiar
4 6
